src/Autocomplete/assets/test/controller.test.ts

```diff
@@ -81,6 +81,7 @@ describe('AutocompleteController', () => {
                 data-testid="main-element"
                 data-controller="check autocomplete"
                 data-autocomplete-url-value="/path/to/autocomplete"
+                data-autocomplete-min-characters-value="3"
```

Comment:
To test this, I would remove this attribute from this test. Then create a new test - e.g. `it limits updates when min-characters` is passed. Inside, you would duplicate this test, but a bit simpler:

A) Set the min-characters to 3
B) "type" only 2 characters into the box
C) `expect(container.querySelectorAll('.option[data-selectable]')).toHaveLength(0);` - make sure 0 options are still shown.

No need to mess with `fetchMock` at all, though if an Ajax request WERE made by mistake, `fetchMock` will cause the test to fail since we were not expecting any Ajax calls. In other words, don't do anything with `fetchMock`, but `fetchMock` will "have your back" in case an accidental Ajax request were made.